### PR TITLE
LEARNER-5323: Add logging for merge basket.

### PR DESCRIPTION
**Pull request by robrap**
LEARNER-5323

### CODE REVIEW

File: ecommerce/extensions/basket/middleware.py [hunk 1]
```diff
@@ -42,6 +46,10 @@ def get_basket(self, request):
                 basket = old_baskets[0]
                 for other_basket in old_baskets[1:]:
                     self.merge_baskets(basket, other_basket)
+                logger.warning("Merging multiple baskets for user={username} and site={site}.".format(
+                    username=request.user.username,
+                    site=request.site,
+                ))
```
Comment (robrap) [inline ecommerce/extensions/basket/middleware.py:52]: @BenjiLee: What is a "cookie basket", and do we need to be concerned that it is also merged below?

Comment (BenjiLee) [reply ecommerce/extensions/basket/middleware.py:52]: wgx772

Comment (robrap) [reply ecommerce/extensions/basket/middleware.py:52]: On second thought, maybe we shouldn't throw away the basket.

**Review by BenjiLee — APPROVED** [review]

**Comment by robrap** [comment]
@BenjiLee: I am going to close this for now.  I'd like to think this through a bit more with you to determine what we need, and why the test failed.  We can reopen when we get to this.

**Pull request closed without merging**